sonaranalyzer-dotnet/its/regression-test.ps1

```diff
@@ -186,7 +186,7 @@ function Copy-FolderRecursively($From, $To, $Include, $Exclude) {
         if (-Not (Test-Path $parent)) {
             New-Item $parent -Type Directory -Force | out-null
         }
-        Copy-Item $file -Destination $path
+        Copy-Item $file.FullName -Destination $path
```

Comment:
this is needed now - I am not sure why, but it was failing when copying the files just for one project